airflow/providers/ssh/hooks/ssh.py

```diff
@@ -131,9 +132,9 @@ def __init__(  # pylint: disable=too-many-statements
                     self.key_file = extra_options.get("key_file")
 
                 private_key = extra_options.get('private_key')
-                private_key_passphrase = extra_options.get('private_key_passphrase')
+                self.private_key_passphrase = extra_options.get('private_key_passphrase')
                 if private_key:
-                    self.pkey = self._pkey_from_private_key(private_key, passphrase=private_key_passphrase)
+                    self.pkey = self._pkey_from_private_key(private_key, passphrase=self.private_key_passphrase)
```

Comment:
```suggestion
                    self.pkey = self._pkey_from_private_key(
                        private_key, passphrase=self.private_key_passphrase
                    )
```
to make pylint happy

Reply:
:heavy_check_mark: Done